### PR TITLE
Rebasing #742

### PR DESCRIPTION
**Pull request by np**

### CODE REVIEW

File: alot/widgets/thread.py [hunk 1]
```diff
@@ -44,12 +44,10 @@ def __init__(self, message, even=True):
...
             thread_tags = message.get_thread().get_tags(intersection=True)
-            outstanding_tags = set(message.get_tags()).difference(thread_tags)
-            tag_widgets = [TagWidget(t, attr, focus_att)
-                           for t in outstanding_tags]
+            displayed_tags = set(message.get_tags()).difference(thread_tags)
         else:
-            tag_widgets = [TagWidget(t, attr, focus_att)
-                           for t in message.get_tags()]
+            displayed_tags = message.get_tags()
+        tag_widgets = [TagWidget(t, attr, focus_att) for t in displayed_tags]
```
Comment (np) [inline alot/widgets/thread.py]: This is only a refactoring now.

**Comment by pazz** [comment]
I am holding back with this until someone fixes this strange newline problem

**Comment by pazz** [comment]
I just tried again and I still get this really annoying behaviour which I cannot explain at all.

```
thread_statusbar = {tid}, itags: {intersection_tags}  ttags: {thread_tags}
```
Half of the threads will have a two line status bar. Can no-one confirm this?

**Comment by pazz** [comment]
Sorry for the delay: would you mind rebasing to current master to push this along?

**Comment by pazz** [comment]
i rebased and fixed my issue. see #1346

**Pull request closed without merging**